files/image_config/updategraph/updategraph

```diff
@@ -18,6 +18,26 @@ reload_minigraph()
     pfcwd start_default
 }
 
+function copy_config_files()
+{
+    for file in $@; do
+        if [ -f /etc/sonic/old_config/${file} ]; then
+            cp /etc/sonic/old_config/${file} /etc/sonic/
```

Comment:
Let's do some logging here. So when minigraph is missing from the new system we could know when it was lost - before upgrade or during updategraph service.

Reply:
Great idea! Done.